### PR TITLE
Replace \dots with \ldots and \cdots where appropriate

### LINKED ISSUE #11602: Replace \dots with \ldots or \cdots
In a few places in the source we have LaTeX math that uses `\dots`. According to https://tex.stackexchange.com/questions/77737/dots-versus-ldots-is-there-a-difference, `\dots` is `\ldots` in raw LaTeX, but with amsmath, it trues to guess if it should use `\ldots` or `\cdots`. 

Fortunately, none of the printers are using `\dots`, but I still think we should be explicit everywhere. Another issue with this is that KaTeX doesn't support `\dots` (https://github.com/Khan/KaTeX/issues/528).

### PR DESCRIPTION
**Pull request by maapxa**
Fixes #11602

### CODE REVIEW

File: doc/cheatsheet/combinatoric_cheatsheet.tex [hunk 1]
```diff
@@ -425,7 +425,7 @@ \subsection{Methods}
 
 
 \verb!coset_factor(g, af=False)!\\
-Return G�s (self�s) coset factorization, \verb!f,! of \verb!g!.
+Return G�s (self�s) coset factorization, \verb!f,! of \verb!g!.
```
Comment (leosartaj) [inline doc/cheatsheet/combinatoric_cheatsheet.tex]: I am not sure if this change should be here.

File: doc/cheatsheet/combinatoric_cheatsheet.tex [hunk 2]
```diff
@@ -458,7 +458,7 @@ \subsection{Methods}
 
 
 \verb!generate_dimino(af=False)!\\
-Yield group elements using Dimino�s algorithm
+Yield group elements using Dimino�s algorithm
```
Comment (leosartaj) [inline doc/cheatsheet/combinatoric_cheatsheet.tex]: Same.

File: doc/cheatsheet/combinatoric_cheatsheet.tex (hunk 1 above)
Comment (maapxa) [inline doc/cheatsheet/combinatoric_cheatsheet.tex]: no, that should not be here; I will revert this

File: doc/cheatsheet/combinatoric_cheatsheet.tex (hunk 2 above)
Comment (maapxa) [inline doc/cheatsheet/combinatoric_cheatsheet.tex]: same, I will revert this

**Comment by leosartaj** [comment]
+1

**Comment by maapxa** [comment]
@asmeurer We were wondering if the dots in the matrix representation should be cdots or ldots. It seems to me that if there are cdots the comma should also be dropped ([wikipedia](https://en.wikipedia.org/wiki/Matrix_(mathematics))).

**Review by leosartaj — APPROVED** [review]
LGTM.

**Comment by leosartaj** [comment]
Thanks @maapxa!

**Pull request merged**